flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/BucketWriter.java

```diff
@@ -350,25 +343,23 @@ public synchronized void close(boolean callCloseCallback)
     try {
       flush();
     } catch (IOException e) {
-      LOG.warn("pre-close flush failed", e);
+      LOGGER.warn("pre-close flush failed", e);
     }
-    boolean failedToClose = false;
-    LOG.info("Closing {}", bucketPath);
+    LOGGER.info("Closing {}", bucketPath);
     CallRunner<Void> closeCallRunner = createCloseCallRunner();
     if (isOpen) {
       try {
         callWithTimeout(closeCallRunner);
         sinkCounter.incrementConnectionClosedCount();
       } catch (IOException e) {
-        LOG.warn(
+        LOGGER.warn(
           "failed to close() HDFSWriter for file (" + bucketPath +
             "). Exception follows.", e);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "). Exception follows." 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)